app/presenters/collection_presenter.rb

```diff
@@ -0,0 +1,24 @@
+class CollectionPresenter
+  include Enumerable
+
+  attr_reader :collection
+  private :collection
+
+  cattr_accessor :presenter
```

Comment:
@timurvafin I didn't quite get it. Could you please clarify, what is the reason for using cattr_accessor for item presenter?